src/frontends/tensorflow/tests/test_models/gen_wrapper.py

```diff
@@ -19,16 +20,14 @@
 if gen_script.endswith('.py'):
     subprocess.run([sys.executable, gen_script, out_folder], env=os.environ)
 elif gen_script.endswith('.pbtxt'):
-    import tensorflow.compat.v1 as tf
-    from google.protobuf import text_format
-
     model_pbtxt = gen_script
-    with open(model_pbtxt, "r") as f:
-        model_name = os.path.basename(model_pbtxt).split('.')[0]
-        graph_def = tf.GraphDef()
-        text_format.Merge(f.read(), graph_def)
-        tf.import_graph_def(graph_def, name='')
-        tf.io.write_graph(graph_def, os.path.join(sys.argv[2], model_name), model_name + '.pb', False)
+    model_name = os.path.basename(model_pbtxt).split('.')[0]
+    dest_path = os.path.join(out_folder, model_name, model_name + '.pbtxt')
+    os.makedirs(os.path.dirname(dest_path), exist_ok=True)
+    try:
+        shutil.copy(model_pbtxt, dest_path)
+    except shutil.SameFileError:
```

Comment:
why don't you overwrite? what if previous file is wrong (copy incomplete, broken, previous version)?

Looks like original implementation overwrites content each run.

Reply:
Good question, it turned out that some tests in Jenkins just place tests files to the same directory when src_path and dest_path match. So we don't do anything with it and handle exception from shutil.
If copy is incomplete, we have below logic that mark successful copying. In case of unsuccessful copying, this marker is not created and build procedure failures.